apps/web/src/components/templates/email-editor/EmailContentCard.tsx

```diff
@@ -27,7 +26,7 @@ export function EmailContentCard({
   const { readonly } = useEnvController();
   const { control, setValue, watch } = useFormContext(); // retrieve all hook methods
   const contentType = watch(`steps.${index}.template.contentType`);
-  const [activeTab, setActiveTab] = useState<string | null>(EDITOR);
+  const activeTab = contentType === 'customHtml' ? CUSTOM_CODE : EDITOR;
```

Comment:
Time ago I created a type `MessageTemplateContentType` that holds `customHtml` and `editor`. I think I should convert it to an enum and be used in situations like here. What would it be your take on that?

Reply:
yeah, I saw that one, but obviously couldn't use it here, so I went the easiest way 🥲 but saw couple more places where it might be useful